test/cli/commands_test.py

```diff
@@ -924,6 +924,17 @@ def test_cli_disable_noqa_flag():
     assert r"L:   5 | P:  11 | L010 |" in raw_output
 
 
+def test_cli_get_config():
+    config = get_config(
+        "test/fixtures/config/toml/pyproject.toml",
+        True,
+        nocolor=False,
+        verbose=0,
+    )
+    assert config.get("nocolor") is True
+    assert config.get("verbose") == 2
```

Comment:
```suggestion
def test_cli_get_config():
    config = get_config(
        "test/fixtures/config/toml/pyproject.toml",
        True,
        nocolor=None,
        verbose=None,
    )
    assert config.get("nocolor") is True
    assert config.get("verbose") == 2
```
See previous review comment for reason